<hivelabtag>

<h4 id="Top">GutFeeling KnowledgeBase</h4>

<body>
	
					We have developed a proof-of-concept gut microbiome monitoring system using a sequencing and analysis pipeline implemented during our previous I-Corps award (see below).
					<br><br>
					We have collected from the individuals enrolled in our study the following: three separate fecal samples for metagenomic sequencing, anthropometric measurements, a diet history questionnaire, gastrointestinal symptoms questionnaires,
					perceived stress questionnaires, physical activity questionnaires, and sleep questionnaires. We have also begun the analysis of fecal samples from the Human Microbiome Project and the associated metadata. The integration of this data into a
					single knowledgebase of comparable samples using our optimized pipeline will provide the real value of our prototype.
                </p>
	<table width=1100>
		<tr>
			<td>			
	<br>
					<b>Objective</b>
					<br>
					The Gut Feeling Knowledgebase (GFKB) is a reference database of human gut microbiomes from both healthy individuals and those diagnosed with a disease or 
					condition. The GFKB is generated by a metagenomic analysis pipeline described in our paper (doi: <a class=reglinkblue href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0206484"
					target="_blank">10.1371/journal.pone.0206484</a>), and includes three tools 
					which are integrated in the HIVE platform. The aim of this database is to catalog bacterial organisms found within the human digestive tract as the lab
					 continues to conduct metagenomic research on various diseases and conditions (e.g. epilepsy and pre-diabetes). Our hope is to identify key organisms 
					 found within the gut as a means to understand how their imbalances may impact human health. Currently, the HIVE lab has documented over 500 bacterial 
					 organisms within this database, and we hope to continue adding more organisms as we proceed with our current project in predicting intervention outcomes
					  of pre-diabetic patients using their relative gut microbiome abundances.
				
					<br><br>
					<b>GFKB downloads</b>
					<br><br>
					<table style="font-size:13px; border-collapse: collapse;" cellpadding=5 width=100% border="1">
						<tr>
							<th>Version</th>
							<th>Content files</th>
							<th>Format</th>
							<th>File size</th>
							<th>Release notes in plain text file</th>
							<th>Date Created</th>
						</tr>

						<tr>
							<td>v1.0</td>
							<td>
							  <a class=reglinkblue href="/docs/RLDA_KI_Analysis.pdf" download="RLDA_KI_Analysis.pdf">RLDA KI Analysis</a>
							
							</td>
							<td>pdf</td>
							<td>393KB</td>
							<td>
								<a>N/A</a>
							</td>
							<td>May 14 2021</td>
						</tr>
						
						<tr>
							<td>v1.0</td>
							<td>
							  <a class=reglinkblue href="../docs/ML_Matlab_Tutorial.pdf" download="ML_Matlab_Tutorial.pdf">ML Matlab Tutorial</a>
							
							</td>
							<td>pdf</td>
							<td>5.2MB</td>
							<td>
								<a>N/A</a>
							</td>
							<td>January 6 2021</td>
						</tr>

						<tr>
							<td>v5.0</td>
							<td>
							  <a class=reglinkblue href="../docs/GFKB_v5_PreDiabetes.csv" download="GFKB_v5_PreDiabetes.csv">GFKB_v5_PreDiabetes.csv</a>
							
							</td>
							<td>csv</td>
							<td>57KB</td>
							<td>
								<a class=reglinkblue href="../docs/GFKB_v5_release_notes.rtf">GutFeeling Knowledge Base Notes v5.0 </a>
							</td>
							<td>January 18 2023</td>
						</tr>
						
						<tr>
							<td>v4.0</td>
							<td>
								<a class=reglinkblue href="../docs/GutFeelingKnowledgeBase-v4-Master_List.csv" download="GutFeelingKnowledgeBase-v4-Master_List.csv">GutFeelingKnowledgeBase-v4-Master_List.csv</a>
								<br>
								<a class=reglinkblue href="../docs/GutFeelingKnowledgeBase-v4-Epilepsy_Data.csv" download="GutFeelingKnowledgeBase-v4-Epilepsy_Data.csv">GutFeelingKnowledgeBase-v4-Epilepsy_Data.csv</a>
							</td>
							<td>csv</td>
							<td>290KB
								<br>99KB</td>
							<td>
								<a class=reglinkblue href="/GutFeelingKnowledgeBaseNotesV4">GutFeeling Knowledge Base Notes v4.0 </a>
							</td>
							<td>March 31 2020</td>
						</tr>

						<tr>
							<td>v3.0</td>
							<td>
								<a class=reglinkblue href="../docs/GutFeelingKnowledgeBase-v3.csv" download="GutFeelingKnowledgeBase-v3.csv">GutFeelingKnowledgeBase-v3.csv</a>
							</td>
							<td>csv</td>
							<td>44K</td>
							<td>
								<a class=reglinkblue href="/GutFeelingKnowledgeBaseNotesV3">GutFeeling Knowledge Base Notes v3.0</a>
							</td>
							<td>Aug 30 2019</td>
						</tr>

						<tr>
							<td>v2.6</td>
							<td>
								<a class=reglinkblue href="../docs/GutFeelingKnowledgeBase-v2.6.csv" download="GutFeelingKnowledgeBase-v2.6.csv">GutFeelingKnowledgeBase-v2.6.csv</a>
							</td>
							<td>csv</td>
							<td>44K</td>
							<td>
								<a class=reglinkblue href="/GutFeelingKnowledgeBaseNotesV2_6">GutFeeling Knowledge Base Notes v2.6</a>
							</td>
							<td>Jul 23 2018</td>
						</tr>


						<tr>
							<td>v2.6</td>
							<td>
								<a class=reglinkblue href="../docs/HumanGutDB-v2.6.fasta" download="HumanGutDB-v2.6.fasta">HumanGutDB-v2.6.fasta</a>
							</td>
							<td>fasta</td>
							<td>549M</td>
							<td>
								<a class=reglinkblue href="/HumanGutDBv2_6">HumanGutDB v2.6 Notes</a>
							</td>
							<td>Jul 23 2018</td>
						</tr>

						<tr>
							<td>v2.0</td>
							<td>
								<a class=reglinkblue href="../docs/GutFeelingKnowledgeBase-v2.0.csv" download="GutFeelingKnowledgeBase-v2.0.csv">GutFeelingKnowledgeBase-v2.0.csv</a>
							</td>
							<td>csv</td>
							<td>248K</td>
							<td>
								<a class=reglinkblue href="/GutFeelingKnowledgeBaseNotesV2">GutFeeling Knowledge Base Notes v2.0</a>
							</td>
							<td>2017</td>
						</tr>

						<tr>
							<td>v2.0</td>
							<td>
								<a class=reglinkblue href="../docs/HumanGutDB-v2.0.fasta" download="HumanGutDB-v2.0.fasta">HumanGutDB-v2.0.fasta</a>
							</td>
							<td>fasta</td>
							<td>533M</td>
							<td>
								<a class=reglinkblue href="/HumanGutDBv2">HumanGutDB v2.0 Notes</a>
							</td>
							<td>2017</td>
						</tr>

						<tr>
							<td>v2.0</td>
							<td>
								<a class=reglinkblue href="../docs/blackList-v2.0.csv" download="blackList-v2.0.csv">blockList-v2.0.csv</a>
							</td>
							<td>csv</td>
							<td>16K</td>
							<td>
								<a class=reglinkblue href="/blackListNotesV2">Black List Notes v2.0</a>
							</td>
							<td>2017</td>
						</tr>

						<tr>
							<td>v2.0</td>
							<td>
								<a class=reglinkblue href="../docs/unalignedContigsGFKB-v2.0.fasta" download="unalignedContigsGFKB-v2.0.fasta">unalignedContigsGFKB-v2.0.fasta</a>
							</td>
							<td> fasta </td>
							<td>3.2G</td>
							<td>
								<a class=reglinkblue href="/unalignedContigsGFKBNotesV2">Unaligned Contigs GFKB Notes</a>
							</td>
							<td>2017</td>
						</tr>

					</table>
                                        <br><br>
					<h5>GutFeeling KnowledgeBase Relative Abundance Profiles</h5>
					<h6 id="graph_level_title"></h6>
					<div id="doughnut_chart"></div>
					<div id="graph_buttons">
						<button type="button" id="species">Species Level</button>
						<button type="button" id="genus">Genus Level</button>
					</div>

					<br><br>
					<b> <a href="/FilteredNT">Filtered nt</a></b>
					<br>
					<br>The Filtered NT dataset is generated by excluding sequences from the whole nucleotide file provided by NCBI, based on whether they have unwanted taxonomy names or any child taxonomy name of these unwanted ones.
					<br><br>
					<b>Metagenomics Pipeline</b>
					<br>
					We use a two-step pipeline for metagenomic analysis; CensuScope and Hexagon. CensuScope is a census-based tool that randomly samples a user-defined number of reads and BLASTs them against a reference DB.
					Our reference database (a filtered version of NTdb) is the NCBI Nucleotide db with all of the sequences lacking a clear taxonomic lineage filtered out. All artificial sequences have been removed either by our automated filter or manually,
					once an artificial sequence is identified during post analysis processing
					Sequences identified by CensuScope are used as references in Hexagon alignments. HIVE-hexagon, a K-mer based aligner, is more sensitive and faster than current standard alignment algorithms. HIVE-hexagon offers a decrease in computational
					cost, memory requirement and time for processing.



					<br><br>
					<b>Publications</b>
					<br>
					<p>Please cite use with one of the following:</p>
					<ol style="list-style-position:inside;">
						<li><a class=reglinkblue href="https://orcid.org/0000-0003-1409-4549
" target="_blank">King CH</a>, Desai H, Sylvetsky AC, <a class=reglinkblue href="http://orcid.org/0000-0001-6897-5419" target="_blank">LoTempio J</a>, Ayanyan S, Carrie J, <a class=reglinkblue href="http://orcid.org/0000-0002-0836-3389" target="_blank">Crandall K</a>, Fochtman B,
							Gasparyan L, Gulzar N, Howell P, Issa N, Krampis K, Mishra L, Morizono H, Pisegna JR, Rao S, <a class=reglinkblue href="http://orcid.org/0000-0001-6369-3703" target="_blank">Ren Y</a>, Simonyan V, Smith K, VedBrat S, Yao M, <a class=reglinkblue
								href="http://orcid.org/0000-0001-8823-9945" target="_blank">Mazumder R.</a> Baseline human gut microbiota profile in healthy people and standard reporting template. PLOS ONE 2019. doi: <a class=reglinkblue
								href="https://doi.org/10.1371/journal.pone.0206484" target="_blank">10.1371/journal.pone.0206484</a></li>
						<br><br>
						<li>Shamsaddini A, Pan Y, Johnson WE, Krampis K, Shcheglovitova M, Simonyan V, Zanne A, Mazumder R. Census-based rapid and accurate metagenome taxonomic profiling. BMC Genomics. 2014;15(1):918. PMID: <a class=reglinkblue
								href="http://www.ncbi.nlm.nih.gov/pubmed/25336203" target="_blank">25232094</a></li>
						<br><br>
						<li>Santana-Quintero L, Dingerdissen H, Thierry-Mieg J, Mazumder R, Simonyan V. HIVE-Hexagon: High-Performance, Parallelized Sequence Alignment for Next-Generation Sequencing Data Analysis. PLOS One. 2014;9(6):e99033. PMID: <a class=reglinkblue
								href="http://www.ncbi.nlm.nih.gov/pubmed/24918764" target="_blank">24918764</a>.</li>
						<br><br>
						<li>Simonyan V and Mazumder R. High-performance Integrated Virtual Environment (HIVE) Tools and Applications for Big Data Analysis. Genes, 2014 Sep 30;5(4): 957-981. PMID: <a class=reglinkblue href="http://www.ncbi.nlm.nih.gov/pubmed/25271953"
								target="_blank">25271953</a>.</li>
					</ol>
					<br>
					<b>Funding</b>
					<br>
					Current/past: NSF, Otsuka, MGPC

					<br><br>

					<b>Acknowledgements</b>
					<br>
					We would like to thank the following individuals for their significant work in curation and annotation of the GFKB:<br>

                    

                    <ul>
                        <li><a class= reglinkblue href="https://orcid.org/0000-0003-0256-9834" target="_blank"><img src="../imglib/orcid_logo.jpg" alt="ORCID"class ="orcid-icon">Stephanie Singleton </a></li>
                        <li><a  class= reglinkblue href="https://orcid.org/0000-0002-1586-1693"target="_blank"><img src="../imglib/orcid_logo.jpg" alt="ORCID"class ="orcid-icon">Lindsay Hopson </a></li>
                        <li><a  class= reglinkblue href="https://orcid.org/0000-0002-6497-0714"target="_blank"><img src="../imglib/orcid_logo.jpg" alt="ORCID"class ="orcid-icon">Jiuge (April) Yang </a></li>
                        <li><a  class= reglinkblue href="https://orcid.org/0000-0003-4888-9673"target="_blank"><img src="../imglib/orcid_logo.jpg" alt="ORCID"class ="orcid-icon">Tyson Dawson </a></li>
                        <li><a  class= reglinkblue href="https://orcid.org/0000-0003-2299-1426"target="_blank"><img src="../imglib/orcid_logo.jpg" alt="ORCID"class ="orcid-icon">Cameron Sabet </a></li>
                        <li><a  class= reglinkblue href="https://orcid.org/0000-0001-5703-5667"target="_blank"><img src="../imglib/orcid_logo.jpg" alt="ORCID"class ="orcid-icon">Yukta Chidanandan </a></li>
                        <li><a  class= reglinkblue href="https://orcid.org/0000-0002-2577-3240"target="_blank"><img src="../imglib/orcid_logo.jpg" alt="ORCID"class ="orcid-icon">Valery Simonyan </a></li>
                        <li><a  class= reglinkblue href="https://orcid.org/0000-0002-0457-7056"target="_blank"><img src="../imglib/orcid_logo.jpg" alt="ORCID"class ="orcid-icon">Nicole Post </a></li>
                        <li><a  class= reglinkblue href="https://orcid.org/0000-0002-9007-8746"target="_blank"><img src="../imglib/orcid_logo.jpg"  alt="ORCID"class ="orcid-icon">Ben Osborne </a></li>
                        <li><a  class= reglinkblue href="https://orcid.org/0000-0001-9721-3181"target="_blank"><img src="../imglib/orcid_logo.jpg" alt="ORCID" class ="orcid-icon">Sophie Halkett </a>
                        </li>
                        <li><a  class= reglinkblue href="https://orcid.org/0000-0003-1181-8118" target="_blank"><img src="../imglib/orcid_logo.jpg" alt="ORCID"class ="orcid-icon">Miguel Mazumder </a></li>
                        <br><br>

                    
                        <!-- Add more curator names as needed -->
                    </ul>
                    
				</span>


			</td>
		</tr>
	</table>
	<br>
	Questions / Comments
	If you have any questions or comments regarding GutFeelingKB, please contact Raja Mazumder (mazumder@gwmail.gwu.edu).
  <script src="../jslib/d3.v5.min.js"></script>
  <script src="../jslib/graphScript.js"></script>
</body>
